client/common/client.c

```diff
@@ -2248,13 +2249,19 @@ BOOL freerdp_client_handle_pen(rdpClientContext* cctx, UINT32 flags, INT32 devic
 	}
 
 	WLog_WARN(TAG, "Invalid pen %" PRId32 " flags 0x%08" PRIx32, deviceid, flags);
+#else
+	WLog_WARN(TAG, "Pen event detected but RDPEI support not compiled in. Recompile with "
+	               "-DWITH_CHANNELS=ON");
+#endif
```

Comment:
Perhaps we should be more specific (`CHANNEL_RDPEI_CLIENT`) as it seems like `-DWITH_CHANNELS=ON` is not enough

Reply:
Ok, done in the same commit, let me know if you prefer to split.
Thanks for the feedback!